partner/partner-management-service/src/main/java/io/mosip/pms/oidc/client/dto/ClientDetail.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+package io.mosip.pms.oidc.client.dto;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class ClientDetail {
+
+    private String id;
+    private String name;
+    private String policyId;
+    private String policyName;
+    private String relayingPartyId;
```

Comment:
Instead it should be called as partnerId? or consider the logged-in user partner Id?